src/features/users/pages/user.js

```diff
@@ -41,72 +41,54 @@ export const UserPage = ({ match }: Props) => {
     pageMounted({ userId })
   }, [userId])
 
-  if (isLoading) return <LoadingView />
   if (isFailed)
     return <ErrorView error={error || "Cannot load. Please, try again later"} />
 
-  const renderFiltered = (filter) => {
-    switch (filter) {
-      case "useful":
-        return (
-          <NamedCardsList
-            title={`Useful cards for ${displayName(user)}`}
-            cards={useful}
-            renderEmpty={() => (
-              <>
-                <Row>
-                  <H3>No useful cards for you?</H3>
-                </Row>
-                <Row>
-                  <Button as={Link} to="/">
-                    Search useful cards
-                  </Button>
-                </Row>
-              </>
-            )}
-          />
-        )
-      case "my":
-      default:
-        return (
-          <NamedCardsList
-            title={`Cards created by ${displayName(user)}`}
-            cards={created}
-            renderEmpty={() => (
-              <>
-                <Row>
-                  <H3>You don&apos;t have cards yet</H3>
-                </Row>
-                <Row>
-                  <Button as={Link} to="/new/card">
-                    Create new card
-                  </Button>
-                </Row>
-              </>
-            )}
-          />
-        )
-    }
-  }
-
   return (
     <UsersCommonTemplate sidebar={<UserInfo user={user} />}>
-      <ZeroTab
-        onClick={() => {
-          setFilter("my")
-        }}
-      >
-        My cards
+      <ZeroTab active={tab === "useful"} onClick={() => setTab("useful")}>
+        Useful
       </ZeroTab>
       |
-      <ZeroTab
-        onClick={() => {
-          setFilter("useful")
-        }}
-      >
-        Favorite
+      <ZeroTab active={tab === "created"} onClick={() => setTab("created")}>
```

Comment:
All string literals (`"useful"`, `"created"`) must be defined statically in one place and reused by imports for safety feature changes

Reply:
This is not a final solution. After redesign, I will return to this code